src/app/utils/LoginValidation.tsx

```diff
@@ -0,0 +1,22 @@
+import z from "zod";
```

Comment:
This file should be under `src/validations` directory. `app` directory is only meant for pages and api's in Next14.